lib/cronofy/auth.rb

```diff
@@ -2,28 +2,76 @@
 
 module Cronofy
   class Auth
-    API_URL = 'https://api.cronofy.com'
     APP_URL = 'https://app.cronofy.com'
+    API_URL = 'https://api.cronofy.com'
+
+    attr_reader :access_token
```

Comment:
Following from that I then wonder what `Cronofy#request` is adding. It's relying on the `params` parameter to handle encoding which probably means requests aren't being made as JSON which is our preferred format.

It'll work, but we maybe want to switch that, and then it needs to be controlled at the individual callsites which makes `Cronofy#request` much less needed.